### PR TITLE
Feature/koa helmet

### PR DESCRIPTION
**Pull request by Nainterceptor**
Checklist

- [x] I've added a mention in CHANGELOG.md

### CODE REVIEW

File: app.mjs [hunk 1]
```diff
@@ -23,16 +25,29 @@ if (config.get('prometheus_exporter')) {
...
+    if (ctx.headers.host && ctx.headers.host.endsWith(host)) {
+      return ctx.headers.host;
+    }
+    return host;
+  };
+}
 app.use(cors({
+  origin,
   exposeHeaders: [HEADER_CONTENT_RANGES, HEADER_ACCEPT_RANGES],
 })); // @todo: fine tune, for security
```
Comment (Scttpr) [inline app.mjs]: should remove @todo here ?

Comment (Nainterceptor) [reply app.mjs]: You're right !

**Comment by Nainterceptor** [comment]
[Review CORS](https://trello.com/c/e4k7L11w/328-review-cors)

**Comment by Nainterceptor** [comment]
[Ajouter koa helmet](https://trello.com/c/giIw12sS/338-ajouter-koa-helmet)

**Pull request merged**